mne/viz/utils.py

```diff
@@ -1422,3 +1473,99 @@ def on_scroll(self, event):
         self.cbar.draw_all()
         self.mappable.set_norm(self.cbar.norm)
         self.cbar.patch.figure.canvas.draw()
+
+
+class SelectFromCollection(object):
+    """Select channels from a matplotlib collection using `LassoSelector`.
+
+    Selected channels are saved in the ``selection`` attribute. This tool
+    highlights selected points by fading them out (i.e., reducing their alpha
```

Comment:
this should be smothing like "highlights selected points by fading other points out", because the selected points get alpha == 1 and the other points `alpha_other`